### DISCUSSION TITLE
urgent Help! Someone claim “GitHub Student Developer Pack” resources from my student account

### DISCUSSION THREAD

**Discussion by shoukatalidev** [community/community#20949] [GitHub Education]
Hello.

Someone claim “GitHub Student Developer Pack” resources from my student account and than remove my email from preference. I don’t know what happened in past how that person access my student account. Today after signup with university email when I applied for student benefits I got an error “email has already been used to obtain an academic discount on GitHub account(s) MuhammadIshaq1998.” To know more about this I have checked my student email & found mentioned username already claim these resouces. I request you to kindly help me to regain these resources.

I have raised this issue with support but its been 6 days & no one in support have time to reply my this query. Now looking for community for help.

**Comment by JJTech0130** [comment]
How would they have claimed your email? I’m pretty sure you have bigger problems than GitHub student discounts if someone can read your school email (so they could verify it)

**Reply by shoukatalidev** [reply]
I don't know exactly what happened but after that incident changed my password as well setup 2-Factor Authentication.

**Comment by MrSarno** [comment]
Hi @emraantufail,

While I know it's not what you want to hear, only GitHub's support team are able to assist you with this.

While the community are able to help with some issues by offering advice, only staff are able to make changes to GitHub accounts - particularly in cases where you lack access to one of the accounts that are pertinent to the issue you are experiencing.

I would suggest not doing anything to try to 'bump' your position in the queue, as with many helpdesk solutions this resets your position in the queue - and this will result in you having to wait even longer. You haven't said anything to give me the impression that you are doing this, but I just thought I'd mention it in case you've been trying to add more information to your support request.

**Reply by MrSarno** [reply]
Oh, and additionally, I would strongly recommend that you edit your e-mail address out of your post.

This is a public forum. You should avoid sharing your account credentials here - **especially** if you are already having account security issues.

**Reply by shoukatalidev** [reply]
> Oh, and additionally, I would strongly recommend that you edit your e-mail address out of your post.
> 
> This is a public forum. You should avoid sharing your account credentials here - **especially** if you are already having account security issues.

Thanks! I did it

**Comment by c9long** [comment]
Sorry to be the bringer of bad news. The Github Education support team frankly does not handle requests quickly and may have missed yours in entirety. I suggest putting a new request in and just pray they see it. Try bumping your ticket by replying to it. If they continue to ignore you then you're just out of luck :(

They don't seem eager to be giving any students access to the developer pack at all, or at least they're doing anything they can to completely ignore support tickets requesting it. From personal experience, the support team only repeats what already exists as answers in the automated system and provides a very lacklustre amount of "support." If you can make do without the student benefits then I would just continue without them. If you must have them, then cross your fingers and your toes cause this team is quite honestly terrible at handling requests.

**Reply by MrSarno** [reply]
I would, respectfully, feel very dubious about supporting this advice.

Almost all companies’ support teams utilise helpdesk software that queues tickets based on how long a person has been waiting. It is also very common for these applications to group multiple messages from the same customer into a single conversation / ticket so that the employee who responds doesn’t miss any of the information provided by that customer.

This all means that bumping a ticket or starting a new one will, **best case scenario**, change nothing - but much more likely, doing so will effectively ‘reset’ your position in the queue. This will artificially raise the amount of time you have to wait very significantly.

**Reply by c9long** [reply]
That's actually a pretty fair point. I managed to get a reply for my own ticket despite "bumping" it by replying to it, not that the reply did anything but re-state what their automated system already does. 

I guess the question I'd now ask is: If it's been quite some time and they still haven't gotten back to you/your ticket, what can you do to get some proper support on Github?

**Reply by MrSarno** [reply]
It’s hard to say… especially with something seasonal like this - a great many people will all suddenly be interested in the Student Developer Pack at around the same time, due to the academic year beginning in many places around the world at about the same time. I don’t know how much GitHub expands their team in anticipation for this, but the queue size is likely to spike dramatically.

I think the first thing I would do is try asking on GitHub Discussions, a couple of relevant subreddits, and other places asking to hear how long others had to wait before hearing back from the team about this. Ultimately, I don’t think a long wait is necessarily abnormal or a sign of something being wrong - and if it’s a matter of simply waiting to reach the top of the queue, there’s not much that can be done.

**Comment by nexuspcs** [comment]
Enable 2FA on all of your email accounts, and make sure that you can enable 2FA via SMS on your GitHub Account. Also, **change your password** for both your email and your Github Accounts asap.

**Reply by nexuspcs** [reply]
can you mark this as solution if it works?

**Comment by FahmiKamarul** [comment]
can I get an update do you get the solution for it
